src/scripts/plugin.js

```diff
@@ -25,8 +25,13 @@ const dataLanguageDictionary = {
 const getLanguage = (editor) => {
   const blockList = editor.document.getElementsByTag('pre');
   let language = 'Python 3';
-  if (blockList.count() !== 0) {
-    language = dataLanguageDictionary[blockList.getItem(0).getAttribute('data-language')];
+  let i = 0;
+  for (i = 0; i < blockList.count(); i += 1) {
+    const codeBlock = (blockList.toArray())[i];
```

Comment:
Perhaps use
```js
const codeBlock = blockList.getItem(i);
```
will be better? Since it will not cast a new array each iteration.

Reply:
Yes I agree, addressed with 7292f59